test/integration/scheduler_perf/util.go

```diff
@@ -252,6 +252,10 @@ func collectHistogramVec(metric string, labels map[string]string, lvMap map[stri
 	q95 := vec.Quantile(0.95)
 	q99 := vec.Quantile(0.99)
 	avg := vec.Average()
+	if math.IsNaN(avg) {
```

Comment:
I'd like to find out the root cause of NaN avg (or other qXX value) and prevent it as early as we can. I suppose it's because `vec.GetAggregatedSampleCount() == 0`. Could you please help confirm? If so, we can enhance and return in `vec.Validate()`.

Reply:
Okay, I didn't know about vec.GetAggregatedSampleCount(). As expected, it seems that `vec.GetAggregatedSampleCount()` always returns 0 when "NaN panic".

> If so, we can enhance and return in vec.Validate().

So, maybe like this↓? If this looks good to you, I'll close this PR and create a PR for this change (+ unit test) instead.

```go
// Validate makes sure the wrapped histograms have all necessary fields set and with valid values.
func (vec HistogramVec) Validate() error {
+	if vec.GetAggregatedSampleCount() == 0 {
+		return fmt.Errorf("empty histogram")
+	}

	bucketSize := 0
	for i, hist := range vec {
		if err := hist.Validate(); err != nil {
			return err
		}
		if i == 0 {
			bucketSize = len(hist.GetBucket())
		} else if bucketSize != len(hist.GetBucket()) {
			return fmt.Errorf("found different bucket size: expect %v, but got %v at index %v", bucketSize, len(hist.GetBucket()), i)
		}
	}
	return nil
}
```

Reply:
> So, maybe like this↓? If this looks good to you

Yes, this is what I meant.

> I'll close this PR and create a PR for this change (+ unit test) instead.

You can update this PR directly instead of creating another PR.

Reply:
@Huang-Wei Updated. 👍